schema/compose-spec.json

```diff
@@ -91,6 +91,12 @@
                 "context": {"type": "string"},
                 "dockerfile": {"type": "string"},
                 "args": {"$ref": "#/definitions/list_or_dict"},
+                "ssh": {
+                  "oneOf": [
+                    {"type": "string"},
```

Comment:
I understand the idea here to allow one-liner `ssh: default` but I wonder we introduce unnecessary complexity with this, and could just define ssh as a `list_or_dict`, which means one can use:
```yaml
ssh: [default]
```
```yaml
ssh: 
  - myproject=~/.ssh/myproject.pem
```
```yaml
ssh: 
  myproject : ~/.ssh/myproject.pem
```
wdyt?

Reply:
sure it allows something more compact, but with the cost for more confusion. If we allow multiple ssh keys being defined, then let's declare this a list. Otherwise we should allow this for all other places we have lists (volumes, networks, ports, ...)